### ISSUE TITLE
Cant resize to fit to me screen resolution.

### ISSUE THREAD

**Issue by brazda** [JupiterBroadcasting/CasterSoundboard#45]
My resolution is 1366x768 and I can only see about a quarter of the application at a time.
![screenshot from 2017-05-20 16-13-48](https://cloud.githubusercontent.com/assets/20069806/26277458/5e119602-3d77-11e7-8ea6-190d00411530.png)

**Comment by ghost** [comment]
Same here. This could be solved by adding scroll bars.

**Comment by guyfawcus** [comment]
Here too, 1366x768 main display on my laptop. Tried changing the sizes of the icons and making them resizeable but it just makes them too small to be usable. @fszymanski's idea about scroll bars is a good one 👍.

Another fix that might be in the pipeline, potentially with #38, would be to get rid of the icons on every tile which would make it a lot easier to have a smaller main window.

#28 is also kind of related.

**Comment by covarianttensor** [comment]
Yeah, I'm going to have to do that as well. Just  be patient with me, as I am working on something else at the moment. I'll start working on castersoundboard again later this week. I also have to cleanup the code and adopt a style, right now the code looks like a mess.

**Comment by Strit** [comment]
Agreed. This issue makes it almost unusable on laptops below 1080p resolution.

**Comment by Oatwalker** [comment]
I realize you're busy but I too would like to mention this is a problem.  Somehow the OpenPandora people solved this problem for [their device](https://pyra-handheld.com/boards/threads/castersoundboard-1-0.80596/).

**Comment by covarianttensor** [comment]
Sorry for the hiatus, I've been busy with other things and haven't had time to work on CasterSoundboard. I have seen the project fork over at open pandora, I'll check out the code in a few days from now and see what I can back port to our repo.

**Comment by yradunchev** [comment]
[CastrerSoundboard_1366x768](https://github.com/yradunchev/CasterSoundboard_1366x768)

Based on OpenPandora' re-work of the original code... I am sorry but I have very limited to none knowledge of qt5... Just needed this brilliant piece of software on my X220... so allowed myself to mess with the code...

**Comment by covarianttensor** [comment]
> CastrerSoundboard_1366x768

Just so you know I have been developing a new version of the sound board that solves this issue. Hopefully it will solve most of the issues people have with different screen resolutions.

Version 2.0 CasterSoundboard coming before the heat death of the universe (I'm an optimist):

Development screenshots:
![screen shot 2019-02-10 at 1 48 18 pm](https://user-images.githubusercontent.com/507320/52540114-d36d0080-2d3a-11e9-9ec7-1213b50dcf81.png)

![screen shot 2019-02-10 at 1 48 30 pm](https://user-images.githubusercontent.com/507320/52540116-d7008780-2d3a-11e9-87a4-969ece868a0a.png)

**Comment by covarianttensor** [comment]
Maybe I should make a dev blog or youtube channel explaining my software and how to make changes to it. Maybe then I would get some code contributions, lol!

**Comment by yradunchev** [comment]
Thank you, Oscar. Great job! Looking forward to see the new version when ready!

**Comment by pcs3rd** [comment]
When is 2.0 getting released?

**Comment by pcs3rd** [comment]
nvm.

**Comment by LudoBoggio** [comment]
Hello, @covarianttensor . Any news about solving this issue, or the 2.0 version ?
Great work, by the way, for RPG sessions.
